scenarios/kill_wf_attempt/kill_wf_attempt.dig

```diff
@@ -0,0 +1,23 @@
+timezone: Asia/Tokyo
+
+
+_export:
+  baseurl: https://api-workflow.treasuredata.com/
+
++run_long_task:
+  py>: script.loop.loop
```

Comment:
Would be nice to comment this is an example long task and should be replaced what customers want.

Reply:
Sounds nice. I'll write a comment to let users know that '+run_long_task' is a just sample.